sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/AbstractInstrumentBuilder.java

```diff
@@ -33,9 +35,13 @@
   AbstractInstrumentBuilder(
```

Comment:
The refactoring in this class ensures all descriptive fields required for a `toString()` implementation are available when constructed.